src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: LAMPy stack monitoring integration
+tags:
+    - New Relic integrations
+    - LAMPy
+    - Linux
+    - Apache
+    - Python
+metaDescription: Use New Relic browser monitoring to get a dashboard with metrics from your Flutter web application.
+
+---
+
+import infrastructureLampyDashboard from 'images/infrastructure_screenshot-full_lampy-dashboard.png'
+
+Our LAMPy integration makes use of the Infrastructure agent, MySQL agent, Apache agent, and Python agent to provide you with a pre-built dashboard with your most important metrics (like response time, CPU utilization, traffic, and login frequencies)
+
+
+<img
+  title="Lampy dashboard"
+  alt="A screenshot depicting the LAMPy prebuilt dashboard"
+  src={infrastructureLampyDashboard}
+/>
+
+<figcaption>
+After setting up our LAMPy integration, we give you a dashboard for your LAMPy web app metrics. 
+</figcaption>
+
+## Install [#install]
+
+### Step 1: Install the infrastructure agent for Linux [#infra-install]
+
+To do this, follow [the infrastructure agent install steps](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux) for the host containing your Flutter app.
```

Comment:
To do this, follow [the infrastructure agent install steps](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux) for the host containing your LAMPy application.